src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -208,26 +208,12 @@ class ThumbnailsPanel extends React.PureComponent {
 
   onWindowResize = () => {
     this.setState({
-      numberOfColumns: this.getNumberOfColumns(),
+      numberOfColumns: this.getNumberOfColumns(this.state.width),
     });
   }
 
-  getNumberOfColumns = () => {
-    const desktopBreakPoint = 640;
-    const { innerWidth } = window;
-    let numberOfColumns;
-
-    if (innerWidth >= desktopBreakPoint) {
-      numberOfColumns = 1;
-    // TODO: use forwardRef to get the width of the thumbnail div instead of using the magic 20px
-    } else if (innerWidth >= 3 * (THUMBNAIL_SIZE + 20)) {
-      numberOfColumns = 3;
-    } else if (innerWidth >= 2 * (THUMBNAIL_SIZE + 20)) {
-      numberOfColumns = 2;
-    } else {
-      numberOfColumns = 1;
-    }
-
+  getNumberOfColumns = width => {
+    const numberOfColumns = Math.min(3, Math.max(1, Math.floor(width / 160)));
```

Comment:
Let's just have this in the `renderThumbnails` instead of a separate function.

Reply:
as discussed on Xodo, lets keep this fxn so we don't have to rewrite the logic as we need to get the number of columns in several places